tests/OSPSuite.Utility.Tests/NotifierSpecs.cs

```diff
@@ -75,24 +95,16 @@ public class MyObject : Notifier
 
       public string Name
       {
-         get { return _name; }
-         set
-         {
-            _name = value;
-            OnPropertyChanged(() => Name);
-         }
+         get => _name;
+         set => SetProperty(ref _name, value, () => Name);
       }
 
       private string _description;
 
       public string Description
       {
-         get { return _description; }
-         set
-         {
-            _description = value;
-            OnPropertyChanged();
-         }
+         get => _description;
+         set => SetProperty(ref _description, value);
```

Comment:
I twicked the base class even more so that we can omit the expression alltogether. Neat